packages/backend/src/api/request.ts

```diff
@@ -128,7 +128,7 @@ export function buildRequest(options: CreateBackendApiOptions) {
       }
 
       // TODO: Parse JSON or Text response based on a response header
-      const isJSONResponse = headers['Content-Type'] === 'application/json';
+      const isJSONResponse = res?.headers && res.headers?.get('Content-Type') === 'application/json';
```

Comment:
Let's use the constant here: https://github.com/clerkinc/javascript/blob/737bd78434bbf0fc3669fa3ad7da59b9e1b2f7e0/packages/backend/src/constants.ts#L33